BitcoinPrediction.py

```diff
@@ -1,13 +1,18 @@
+# Imports
 import numpy as np
 import pandas as pd
 import tensorflow as tf
 import seaborn as sns
 import matplotlib.pyplot as plt
 
+#Constants
+
```

Comment:
I am a bit of a perfectionist. Could you add appropriate spaces between title and hashtag please :)